src/ray/object_manager/pull_manager.h

```diff
@@ -194,9 +216,10 @@ class PullManager {
   /// Returns whether the request was successfully activated. If this returns
   /// false, then there are no more requests in the queue that can be activated
   /// (because we have reached the end of the queue or because there is missing
-  /// size information).
+  /// size information), or activating the request would exceed memory quota.
```

Comment:
Can you specify that we will allow exceeding the memory quota to maintain at least 1 activate bundle?